scripts/build_test.py

```diff
@@ -721,4 +722,74 @@ def mock_require_compiled_js_dir_to_be_valid():
                 self.assertFalse(
                     os.path.exists(os.path.dirname(MOCK_COMPILED_JS_DIR)))
 
+    def test_build_with_prod_env(self):
+        def mock_build_using_webpack():
+            pass
+
+        def mock_ensure_files_exist(unused_filepaths):
+            pass
+
+        def mock_compile_typescript_files(unused_project_dir):
+            pass
+
+        ensure_files_exist_swap = self.swap(
+            build, '_ensure_files_exist', mock_ensure_files_exist)
+        build_using_webpack_swap = self.swap(
+            build, 'build_using_webpack', mock_build_using_webpack)
+        compile_typescript_files_swap = self.swap(
+            build, 'compile_typescript_files', mock_compile_typescript_files)
+        args_swap = self.swap(sys, 'argv', ['build.py', '--prod_env'])
+
+        with ensure_files_exist_swap, build_using_webpack_swap, (
```

Comment:
why are the () necessary?

Reply:
The parenthesis are required for using multiple lines with the "with" clause, else we would get an `attribute error: __exit__`